arbitrum-docs/how-arbitrum-works/inside-anytrust.md

```diff
@@ -8,48 +8,48 @@ AnyTrust is a variant of Arbitrum Nitro technology that lowers costs by acceptin
 
 ## Data Availability Certificates
 
-A central concept in AnyTrust is the Data Availability Certificate (hereafter, a "DACert"). A DACert contains:
+A central concept in AnyTrust is the Data Availability Certificate (hereafter, a "DACert"). A [DACert](https://github.com/Layr-Labs/nitro/blob/eigenda-v3.2.1/arbstate/daprovider/util.go#L239-L246) contains:
 
 - the hash of a data block, and
 - an expiration time, and
-- proof that N-1 Committee members have signed the (hash, expiration time) pair, consisting of
+- proof that N-1 Committee members have signed the (hash, expiration time) pair, consisting of:
   - the hash of the Keyset used in signing, and
   - a bitmap saying which Committee members signed, and
-  - a BLS aggregated signature (over the BLS12-381 curve) proving that those parties signed.
+  - an aggregated BLS signature (over the BLS12-381 curve) proving that those parties signed.
 
-Because of the 2-of-N trust assumption, a DACert constitutes proof that the block's data (i.e., the preimage of the hash in the DACert) will be available from at least one honest Committee member, at least until the expiration time.
+Because of the `1-of-N` trust assumption, a DACert constitutes proof that the batch data (i.e., the preimage of the hash in the DACert) will be available from at least one honest Committee member, at least until the expiration time.
 
-In ordinary (non-AnyTrust) Nitro, the Arbitrum sequencer posts data blocks on the L1 chain as calldata. The hashes of the data blocks are committed by the L1 Inbox contract, allowing the data to be reliably read by L2 code.
+The hashes of the data blocks are committed by the L1 chain's Sequencer Inbox contract, allowing the data to be reliably read by L2 chain nodes.
 
-AnyTrust gives the sequencer two ways to post a data block on L1: it can post the full data as above, or it can post a DACert proving availability of the data. The L1 inbox contract will reject any DACert that uses an invalid Keyset; the other aspects of DACert validity are checked by L2 code.
+AnyTrust gives the sequencer an additional way to post batches to the L1 chain. The L1 Sequencer Inbox contract will reject any DACert that uses an invalid Keyset; other certificate validity checks are enforced during derivation. 
 
-The L2 code that reads data from the inbox reads a full-data block as in ordinary Nitro. If it sees a DACert instead, it checks the validity of the DACert, with reference to the Keyset specified by the DACert (which is known to be valid because the L1 Inbox verified that). The L2 code verifies that
+The L2 code that reads data from the inbox reads a full-data block as in ordinary Nitro. If it sees a DACert instead, it checks the validity of the DACert, with reference to the Keyset specified by the DACert (which is known to be valid since the Sequencer Inbox verified that). The L2 chain code verifies that:
```

Comment:
SequencerInbox is usually written without spaces.

We should be consistent with the language. `L2 code` is a valid expression, but if we want to change it to another thing, we would have to change all instances. 

In any case, my recommendation for an external contribution is not to change language that is correct, even if there might be a better way to express that concept, since we take a global approach within the docs, and try to use the same language across all pages. The best way to deal with changing language is to open an issue so we can discuss internally how to approach the change 🙏